FESS-GUI/flypacketlib.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FLYPACKETLIB_H
+#define FLYPACKETLIB_H
+
+#include "datatypes.h"
+
+flypacket buildFlyPacket(flybyte);
```

Comment:
I think flypacket should be a class, with the same members as the struct has now, and this function as a constructor. This would make for a more object-oriented design.